docs/patterns/card/patterns/asset-title-and-link-top.html

```diff
@@ -20,9 +29,7 @@ <h4>Title, Lg</h4>
       margin-block: 0 var(--rh-space-lg, 16px);
       display: flex;
       gap: var(--rh-space-lg, 16px);
-      font-weight: var(--rh-font-weight-heading-regular, 300);
-      font-size: var(--rh-font-size-body-text-md, 1rem);
-      font-family: var(--rh-font-family-body-text);
+      color: var(--rh-color-text-brand-on-light, #ee0000);
```

Comment:
@marionnegp can you confirm that we'd need to use the theming token `light-dark()` variation here: `--rh-color-text-brand`.

```css
var(--rh-color-brand-red, 
  light-dark(
    var(--rh-color-brand-red-on-light, #ee0000), 
    var(--rh-color-brand-red-on-dark, #ee0000)
  )
);
```